src/Relude_OptionT.re

```diff
@@ -26,6 +26,10 @@ module WithMonad = (M: BsAbstract.Interface.MONAD) => {
     (aToB, OptionT(mOptionA)) =>
       OptionT(M.map(optionA => Relude_Option.map(aToB, optionA), mOptionA));
 
+  let semiFlatMap: 'a 'b. ('a => option('b), t('a)) => t('b) =
```

Comment:
I was looking at cats to refresh my memory on `semi`/`subflatMap`, and I'm wondering if this signature is actually `subFlatMap`?  I recall using `subflatMap` in the past in cats, and it made sense to me b/c you're flatMapping the inner/sub monad.

https://github.com/typelevel/cats/blob/bfc264eddd621bc81dff681060be37656e04f72b/core/src/main/scala/cats/data/OptionT.scala#L64-L65

I would love to have both `sub` and `semiFlatMap` if you agree and would like to rename this to `subFlatMap`, and add `semiFlatMap`?  Or let me know if you disagree with this - I might be just confused.